doc/api/esm.md

```diff
@@ -437,8 +437,8 @@ The `--experimental-json-modules` flag is needed for the module
 to work.
 
 ```bash
-node --experimental-modules --entry-type=module index.js # fails
-node --experimental-modules --entry-type=module --experimental-json-modules index.js # works
+node --experimental-modules index.js # fails
+node --experimental-modules --experimental-json-modules index.js # works
```

Comment:
```suggestion
node --experimental-modules --experimental-json-modules index.mjs # works
```

Reply:
(Also need to update line 429 with the right file extension too.)

Reply:
good point.